wdqs/0.3.1/wait-for-it.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+# This file is provided by the wikibase/wdqs docker image.
+# Use this script to test if a given TCP host/port are available
+
+cmdname=$(basename $0)
+
+echoerr() { if [[ $QUIET -ne 1 ]]; then echo "$@" 1>&2; fi }
```

Comment:
You could also make the function definition conditional, to make it a bit more efficient when executed:

```sh
if [[ $QUIET != 1 ]]; then
    echoerr() {
        printf >&2 '%s\n' "$1"
    }
else
    echoerr() { :; }
fi
```

(Also includes some other improvements – use `!=` instead of `-ne` in `[[`, use `printf` instead of `echo` to avoid escape sequence interpretations)